modules/_node-scripts/configuration/getProjectMain.mjs

```diff
@@ -0,0 +1,10 @@
+import require from '../util/require.mjs';
+
+/**
+ * @returns the proejct relative path of the main entry point
+ */
+export default function getProjectExports() {
```

Comment:
name should be `getProjectMain`?

Reply:
:scream:  End of the world!!!

![end of the world](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExb2N5bzdocHN2MnJvNG10bm1raDY0czh0Y3huazBnczJjMDl3ZDV4NCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/Iep1piW2Na91q0gWYX/giphy-downsized-large.gif)

Reply:
done